docs/sdk.rst

```diff
@@ -169,58 +169,73 @@ latter also works with the ``Executor`` class.
 Batching
 --------
 
-The SDK includes a batch interface to reduce the overheads of launching a function many times.
-To use this interface, you must first create a batch object and then pass that object
-to the ``batch_run`` function. ``batch_run`` is non-blocking and returns a list of task ids
-corresponding to the functions in the batch with the ordering preserved.
+The SDK includes a batch interface to reduce the overhead of launching a function many
+times.  To use this interface, first create a |Batch| object and then pass that
+object to the |batch_run()| method.  The |batch_run()| method only blocks for the
+duration of the API call; the tasks may not yet have been received by the endpoint when
+it returns.
```

Comment:
A few more words as to that that means?  You removed the text about it returning task ids without adding something similar.  (ie. the task IDs returned can be used to track the tasks but not right away as the EP may not recognize)

Reply:
*nod; I moved it to above the second fragment.  Based on this feedback, however, I've put it back in place, and joined the code fragments.  How does this read to you now?